resources/views/layouts/base.blade.php

```diff
@@ -36,6 +36,7 @@
 
 @include('layouts._ads')
 <script src="{{ mix('js/app.js') }}" defer></script>
+@livewireAssets
```

Comment:
I think this needs to go into the `<head>` tag at the end?

Reply:
`@livewireAssets` needs to be included after `app.js` otherwise they conflict. Happy to move both the head if you prefer?

Reply:
It's okay 👍